Makefile

```diff
@@ -20,26 +20,22 @@ OPS_BIN_TARGET = ./build/bin/genesis # relative to OPS_DIR
 OPS_BIN_SRC = ./cmd/genesis/ # relative to OPS_DIR
 OPS_BINDINGS_TARGET = $(OPS_DIR)/bindings
 
+BINDINGS_TARGET = bindings-go
+
 # TODO add clef back in when moving to services/el_clients/go-ethereum
 #CLEF_SRC = $(SIDECAR_DIR)/cmd/clef/
 #CLEF_TARGET = $(SIDECAR_BIN)/clef
 
 install: geth magi sidecar ops
-geth: ops-bindings $(GETH_BIN_TARGET)
+geth: bindings $(GETH_BIN_TARGET)
 magi: $(MAGI_BIN_TARGET)
-sidecar: bindings $(shell find $(SIDECAR_DIR) -type f -name "*.go")
+sidecar: $(shell find $(SIDECAR_DIR) -type f -name "*.go")
```

Comment:
it still needs the bindings right?

Reply:
yes, I overlooked this change from the earlier PR - good spot